examples/tls/cert/ca.crt

```diff
@@ -0,0 +1,3 @@
+-----BEGIN CERTIFICATE-----
+<pem content>
```

Comment:
Please replace <pem content> with the content of the CA certificate actually generated, which is the last paragraph in server.crt

Reply:
fixed